reference/catalog/custom-fields.v2.yml

```diff
@@ -0,0 +1,390 @@
+openapi: '3.0.3'
```

Comment:
@olysenko we do not currently provide any swagger for the Catalog V2 API. let's discuss with y'all, @bc-tgomez, and myself.